examples/langchain-ollama/02-customer_email.py

```diff
@@ -0,0 +1,53 @@
+from humanlayer import ContactChannel, SlackContactChannel
+from langchain.agents import AgentType, initialize_agent
+import langchain_core.tools as langchain_tools
+from langchain_openai import ChatOpenAI
```

Comment:
this import breaks because it's not in the requirements.txt (and there are a few other imports here that are unused)

if you remove them you should be good to go